frontend/src/components/commons/Navbar.tsx

```diff
@@ -35,29 +45,40 @@ const Navbar: FC = () => {
           </NavLink>
         </li>
       </ul>
+
       <ul className="flex flex-row justify-between p-10 w-100 mt-auto">
-        <li className="p-5">
-          <NavLink exact to={`/login`} activeClassName="text-gold-200">
-            Login
-          </NavLink>
-        </li>
-        <li className="p-5">
-          <NavLink exact to={`/register`} activeClassName="text-gold-200">
-            Register
-          </NavLink>
-        </li>
-        {/* To-do: Authentication checking
-          <li className="p-5">
-            <NavLink to={`/account`} activeClassName="text-gold-200">
-              Account
-            </NavLink>
-          </li>
-          <li className="p-5">
-            <NavLink to={`/logout`} activeClassName="text-gold-200">
-              Logout
-            </NavLink>
-          </li>
-        */}
+        {!user && (
+          <>
+            <li className="p-5">
+              <NavLink exact to={`/login`} activeClassName="text-gold-200">
+                Login
+              </NavLink>
+            </li>
+            <li className="p-5">
+              <NavLink exact to={`/register`} activeClassName="text-gold-200">
+                Register
+              </NavLink>
+            </li>
+          </>
+        )}
+
+        {user && (
+          <>
+            <li className="p-5">
+              <NavLink to={`/account`} activeClassName="text-gold-200">
+                Account
+              </NavLink>
+            </li>
+            <li className="p-5">
+              <div className="text-gold-200" onClick={handleOnClick}>
+                Logout
+              </div>
+              {/* <NavLink to={`/logout`} activeClassName="text-gold-200">
+                Logout
+              </NavLink> */}
```

Comment:
Since you make this `div` can clickable, remove below comment and add `role="button"` to it attribute for ARIA for more information see [https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role)

Reply:
@Icyscools Yes, I should make it a `button` or put `role="button"`